core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/BlockCacheManager.java

```diff
@@ -30,7 +30,7 @@
 
   private final Map<CacheType,BlockCache> caches = new HashMap<>();
 
-  public static final String CACHE_PROPERTY_BASE = Property.GENERAL_ARBITRARY_PROP_PREFIX + "cache.";
+  public static final String CACHE_PROPERTY_BASE = Property.TSERV_PREFIX + "cache.";
```

Comment:
Inorder to avoid system cache properties from conflicting with custom cache config properties, I am thinking it would be better to make this `Property.TSERV_PREFIX + "cache.config."`.

Reply:
I actually had that before, I will change it and rebuild and then hopefully it works with keith-turner/accumulo-ohc .